rollup.config.js

```diff
@@ -67,10 +76,10 @@ module.exports = [
 	},
 
 	// ES6 builds
-	// dist/chart.esm.min.js
 	// dist/chart.esm.js
```

Comment:
I was wondering if the `dist/` needs to be removed from this line how that we have `dir: './'` instead of `file: 'dist/chart.esm.js'`?

Reply:
It doesn't need to, package.json has: `"module": "dist/chart.esm.js"`.

Reply:
Oh, I see the output directory is specified in `inputESM` now, which is what I missed earlier. I wonder though if we should output it to the root directory? It seems a bit inconsistent that the main bundle and helpers are in different directories

Reply:
Its easier to define the `files` in `package.json` this way.
Looks like the `helpers` can be aliases under dist too, I'll do that  instead.

Reply:
> Its easier to define the `files` in `package.json` this way.
> Looks like the `helpers` can be aliases under dist too, I'll do that instead.

That one did not work with rollup or webpack, so reverted.

Reply:
I'm curious why it's harder to define the `files` as being the root directory. Is it because it's picking up the `karma.conf.js` and `rollup.config.js` then or something? I don't think there's any harm in that, but if you want to avoid it we could delete them in deploy script

Reply:
```js
files: [
  "chart.esm.js", // need to add this line -> harder :D
  "dist/*.js",
  "helpers/**.js"
]
```

Reply:
If we start moving files out of dist just because the helpers need to be in root, we might as well delete the dist folder and put everything in root.

Are there any good reasons to move things to root? The `src` -> `dist` is a common pattern. Just the `exports` in package.json does not seem to be supported widely yet. (it would allow having the helpers in dist too and still using `import {close} from 'chart.js/helpers/canvas'`)

Now one other common practice is to just publish the dist folder, but that requires copying package.json etc. there. Not a fan of copying stuff.

Reply:
Yeah, I was basically suggesting getting rid of `dist/` entirely. We could have a production mode where rollup just outputs to the root directory to avoid having to copy or move the output.

I've seen some projects use dist and some use the root directory. I'm fine with either. I just think it's a little weird to split between the two. But since it doesn't have any practical difference I won't fuss over it

Reply:
Figured that much. We have the samples and docs that go through dist too, so IMO it would be just wasted effort to change those.